pkg/istio-agent/sds-agent.go

```diff
@@ -430,6 +430,9 @@ func applyEnvVars() {
 	workloadSdsCacheOptions.SecretTTL = secretTTLEnv
```

Comment:
Can we also default the grace period and refresh interval based on this one? Otherwise its so easy to shoot yourself in the foot if you don't configure all 3 options